draft-ietf-scitt-architecture.md

```diff
@@ -404,19 +402,20 @@ In SCITT Statements and Receipts, the `iss` CWT Claim is a member of the COSE he
 
 Non-equivocation:
 
-: a state where it is impossible for a Transparency Service to provide different views of its Append-only Log to Relying Parties about the same Artifact.
+: a state where it is impossible for a Transparency Service to provide different Verifiable Data Structures to Relying Parties about the same Artifact.
```

Comment:
```suggestion
: a state where all Verifiable Data Proofs provided by the Transparency Service to Relying Parties are produced from a Single Verifiable Data Structure describing a unique sequence of Signed Statements, and are therefore compatible.
```

Similar to the clarification above, probably a bit heavy handed, but trying to avoid stating that the VDS is provided to RPs.

Reply:
```suggestion
: a state where all proofs provided by the Transparency Service to Relying Parties are produced from a Single Verifiable Data Structure describing a unique sequence of Signed Statements, and are therefore consistent.
```

avoid adding "Verifiable Data Proofs" and just rely on mention of VDS for context ?
Also the nit: compatible/consistent